Peer.Connectors/AzdoPullRequestSource.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.TeamFoundation.Core.WebApi;
+using Microsoft.TeamFoundation.SourceControl.WebApi;
+using Peer.Domain;
+using Peer.Domain.Models;
+using Peer.User;
+
+namespace Peer.Connectors
+{
+    public class AzdoPullRequestSource : IPullRequestSource<AzdoPullRequestSource>
+    {
+        private readonly AppConfig _config;
+        private static IClient<UserAzdoClient> _userClient;
+
+        public AzdoPullRequestSource(AppConfig config, IClient<UserAzdoClient> userClient)
+        {
+            _config = config;
```

Comment:
`_config` doesn't seem to be used at all here. I haven't seen the `IClient` stuff yet but does `UserAzdoClient` already have the auth/id info baked in or something?